targets/TARGET_STM/TARGET_STM32F4/analogin_api.c

```diff
@@ -183,13 +183,14 @@ static inline uint16_t adc_read(analogin_t *obj)
             return 0;
     }
 
+    ADC->CCR &= ~(ADC_CCR_VBATE | ADC_CCR_TSVREFE); // Workaround
```

Comment:
Workaround? I dont see any explanation in the commit message neither here. What does this solve?  can you add more details to the commit message?